### PR TITLE
Attempt to fix null current Namespace bug

### PR DESCRIPTION
**Pull request by callen-sfdo**
An attempt to fix the intermittent bug that occurs when the namespaceWrapper is not defined in time for the getAlreadyMappedDIFields method in the bdiObjectMappingModal to use it.  A second PR will be created to remove the debug statements.

# Critical Changes

# Changes

# Issues Closed

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js [hunk 1]
```diff
@@ -526,9 +529,10 @@ export default class bdiObjectMappingModal extends LightningElement {
     refreshImportRecordFieldOptions() {
         this.diImportRecordFieldOptions = [];
         this.diImportRecordStatusFieldOptions = [];
-
+        console.log('in refreshImportRecordFieldOptions namespace Wrapper is: ' + this.namespaceWrapper);
         let fieldsToExclude = this.getAlreadyMappedDIFields(); 
-        if (fieldsToExclude && this.dataImportFieldData) {
+        
+        if (fieldsToExclude && this.dataImportFieldData && this.namespaceWrapper) {
```
Comment (ptewson-sfdo) [inline src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js]: Doesn't the call to this.getAlreadyMappedDIFields() on line 533 also reference "this.namespaceWrapper" and it's also outside of this if statement? There were a couple variations of the error message that showed up, and the one that reads "cannot read property 'currentNamespace' of undefined" I think might not be protected by the new conditional.

**Comment by ptewson-sfdo** [comment]
**lurch: attach W-035852

File: src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js [hunk 2]
```diff
@@ -172,6 +172,7 @@ export default class bdiObjectMappingModal extends LightningElement {
             this.getDataImportFieldDescribes();
             this.getDataImportMappedFields();
         } catch(error) {
+            console.log('Error encountered in objectMappingModal init: ' + error);
```
Comment (ptewson-sfdo) [inline src/lwc/bdiObjectMappingModal/bdiObjectMappingModal.js]: For outside observers: these logs are temporary and will be removed, we just need to be able to track down the nature of the issue if our first fix doesn't tackle it. These logs will minimize future iterative merges to master.

**Comment by LurchTheButler** [comment]
Tracking <a href="https://foundation.my.salesforce.com/a2x1E000001CAPrQAO" target="_blank">W-035852</a>

**Pull request merged**